<<<<<<< HEAD
<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchResult</class>
 <widget class="QDialog" name="SearchResult">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1045</width>
    <height>513</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>311</width>
     <height>401</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>50</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255,255,255);
font: 12pt &quot;微软雅黑&quot;;
background-color: rgb(255, 185,0);
border-radius:15px;
</string>
   </property>
   <property name="text">
    <string>    相关问题</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget_2">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>90</y>
     <width>301</width>
     <height>401</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget_3">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>90</y>
     <width>311</width>
     <height>401</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>50</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255,255,255);
font: 12pt &quot;微软雅黑&quot;;
background-color: rgb(255, 185,0);
border-radius:15px;
</string>
   </property>
   <property name="text">
    <string>    相关回答</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>50</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255,255,255);
font: 12pt &quot;微软雅黑&quot;;
background-color: rgb(255, 185,0);
border-radius:15px;
</string>
   </property>
   <property name="text">
    <string>    相关用户</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
=======
<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchResult</class>
 <widget class="QDialog" name="SearchResult">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1045</width>
    <height>513</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>311</width>
     <height>401</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>50</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255,255,255);
font: 12pt &quot;微软雅黑&quot;;
background-color: rgb(255, 185,0);
border-radius:15px;
</string>
   </property>
   <property name="text">
    <string>    相关问题</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget_2">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>90</y>
     <width>301</width>
     <height>401</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget_3">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>90</y>
     <width>311</width>
     <height>401</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>50</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255,255,255);
font: 12pt &quot;微软雅黑&quot;;
background-color: rgb(255, 185,0);
border-radius:15px;
</string>
   </property>
   <property name="text">
    <string>    相关回答</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>50</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255,255,255);
font: 12pt &quot;微软雅黑&quot;;
background-color: rgb(255, 185,0);
border-radius:15px;
</string>
   </property>
   <property name="text">
    <string>    相关用户</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
>>>>>>> cfdf638c3bdfa8efdbbb0c911ad54fe243c63989
